.pre-commit-config.yaml

```diff
@@ -17,14 +17,11 @@ repos:
     hooks:
       - id: pyupgrade
         args: ["--py38-plus"]
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.269
     hooks:
-      - id: isort
-        name: isort (python)
-      - id: isort
-        name: isort (pyi)
-        types: [pyi]
+      - id: ruff
+        args: ["--fix", "--fixable=I001", "--exit-non-zero-on-fix"]
```

Comment:
`--exit-non-zero-on-fix` is suggested at https://github.com/charliermarsh/ruff-pre-commit